Kitodo/src/main/webapp/WEB-INF/resources/css/kitodo.css

```diff
@@ -23,9 +23,12 @@ body {
     line-height: 18px;
     background: url("#{resource['/images/background-top.svg']}") no-repeat #f3f3f3 center top fixed;
     background-size: auto 300px;
+    }
+
+body.first-visit {
     animation-duration: 1s;
     animation-name: slidein;
-    }
+}
```

Comment:
Is this indent correct? In whole file the closing bracket is indent by 4 spaces.